### PR TITLE
DEP 4.11 dev requirements

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripeltd/product-issues/issues/531

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -7,18 +7,18 @@
...
+        "silverstripe/admin": "1.11.x-dev",
+        "silverstripe/asset-admin": "1.11.x-dev",
+        "silverstripe/campaign-admin": "1.11.x-dev",
+        "silverstripe/versioned-admin": "1.11.x-dev",
+        "silverstripe/cms": "4.11.x-dev",
+        "silverstripe/errorpage": "1.11.x-dev",
+        "silverstripe/reports": "4.11.x-dev",
+        "silverstripe/siteconfig": "4.11.x-dev",
+        "silverstripe/versioned": "1.11.x-dev",
+        "silverstripe/graphql": "4.0.x-dev",
```
Comment (GuySartorelli) [inline composer.json:20]:
```suggestion
        "silverstripe/graphql": "3.8.x-dev  || 4.0.x-dev",
```
We're still supporting both versions of graphql

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (emteknetnz) [reply composer.json:20]: We do, however in order to use graphql 3 you need to 'inline' (stop using) recipe-cms

Comment (GuySartorelli) [reply composer.json:20]: Ahh yup, good point. I do remember seeing that in the doc.

**Review by GuySartorelli — APPROVED** [review]
LGTM. Merge on green.

**Pull request merged**